src/Text/Parsing/Parser/Token.purs

```diff
@@ -77,52 +76,66 @@ type LanguageDef = GenLanguageDef String Identity
 -- | features of the "Text.Parsec.Token" module. The module `Text.Parsec.Language`
 -- | contains some default definitions.
 newtype GenLanguageDef s m
-    = LanguageDef {
+  = LanguageDef
```

Comment:
@natefaubion This one is a bit weird; ideally, the commas would be lined up with the values in the record themselves, though I understand that the original formatting with trailing commas is the cause of the issue.

Reply:
The formatter doesn’t relocate comments to other tokens. In fact, it tried to preserve it as much as possible. That’s why it keeps the newline after the comma, so that the comment is preserved as a leading comment on that token, and not relocated to a trailing comment on the comma.